src/identity/components/OrganizationListTab/OrgList.tsx

```diff
@@ -112,6 +112,7 @@ export const OrgList: FC<Props> = ({
           provider={org.provider}
           regionCode={org.regionCode}
           regionName={org.regionName}
+          provisioningStatus={org.provisioningStatus}
         />
```

Comment:
ah good point. I was under the impression we always get back a `status`

Reply:
if I'm following correctly, are you suggesting to only check if `provisioningStatus` is true (aka value has been passed in) instead of explicitly checking if `provisioningStatus === 'suspended` to render or not render the `Deletion in progress`  text inside `OrganizationCard.tsx`?

Reply:
The org should always have a `provisioningStatus`, so it's okay to not worry about this.